templates/errors/submission-complete.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/_base.html' %}
+
+{% set page_title = _("Submission Complete") ~ " - " ~ _("Census 2021") %}
+
+{% block main %}
+<h1>{{ _("This page is no longer available") }}</h1>
```

Comment:
Again here, also the page title makes sense, it would be good to check where we can keep it as is, or the title should match h1.